docs/compute/modules.md

```diff
@@ -88,7 +88,7 @@ Command                 | Shortened Command            | Description  | Example
 ## Loading Modules in a Job Script
```

Comment:
On this page there are parts saying to do `module load python/3.5.1`, I would replace this with `module load anaconda`. Also in the table in the `Subcommands` section there is a part that is loading GCC 6.1.0. This GCC isn't on our system. Please update it with a GCC that is on our system.

Reply:
I am still seeing a Python there. Additionally, the change you made for the code below:
```
You can specify the version of the software by appending a / with the version number:
```
does not make sense. 

In the `Loading Modules in a Job Script` section it states `A sample job script that loads Python into the environment is provided below:`. However, you are loading in anaconda (also replace `python3` with `python`).